### PR TITLE
Fixes suggested by pylint

### PR DESCRIPTION
**Pull request by IvarsKarpics**
Cleaned up base classes and bricks based on pylint:
* removed not used code.
* added missing items to __init__
* removed not used imports
* etc

### CODE REVIEW

File: gui/bricks/AttenuatorsBrick.py [hunk 1]
```diff
@@ -92,30 +93,51 @@ def __init__(self, *args):
         self.instance_synchronize("transmission_ledit", "new_value_ledit")
 
     def run(self):
+        """Init api and start gui"""
         if api.transmission is not None:
-            self.connect(api.transmission, "deviceReady", self.connected)
-            self.connect(api.transmission, "deviceNotReady", self.disconnected)
-            self.connect(api.transmission, "stateChanged", self.transmission_state_changed)
-            self.connect(api.transmission, "valueChanged", self.transmission_value_changed)
+            self.connect(api.transmission,
```
Comment (rhfogh) [inline gui/bricks/AttenuatorsBrick.py]: This does not look like what black would suggest

File: gui/bricks/AttenuatorsBrick.py [hunk 2]
```diff
@@ -92,30 +93,51 @@ def __init__(self, *args):
...
             Colors.set_widget_color(
-                self.new_value_ledit, Colors.LINE_EDIT_CHANGED, QtImport.QPalette.Base
+                self.new_value_ledit,
+                Colors.LINE_EDIT_CHANGED,
+                QtImport.QPalette.Base
             )
         else:
             Colors.set_widget_color(
-                self.new_value_ledit, Colors.LINE_EDIT_ERROR, QtImport.QPalette.Base
+                self.new_value_ledit,
```
Comment (rhfogh) [inline gui/bricks/AttenuatorsBrick.py]: Again, are w2e not supposed ot use black formatting?

File: gui/bricks/EMBL/StillCollectionPreviewBrick.py [hunk 3]
```diff
@@ -153,104 +152,85 @@ def __init__(self, *args):
...
-                    "processingFailed", self.processing_failed
-                )
-                self.parallel_processing_hwobj.disconnect(
-                    "processingFrame", self.processing_frame_changed
-                )
-            self.beamline_setup_hwobj = self.get_hardware_object(new_value)
-            self.parallel_processing_hwobj = (
-                self.beamline_setup_hwobj.parallel_processing_hwobj
-            )
+        api.parallel_processing.connect("processingStarted", self.processing_started)
```
Comment (rhfogh) [inline gui/bricks/EMBL/StillCollectionPreviewBrick.py:155]: Is api.parallel_processing always set?

File: gui/bricks/MachineInfoBrick.py [hunk 4]
```diff
@@ -80,15 +80,14 @@ def set_value(self, values_list):
            At first time initializes gui by adding necessary labels.
            If the gui is initialized then update labels with values
         """
-
         if self.graphics_initialized is None:
-            for item in values_list:
+            for item in values_list.values():
```
Comment (rhfogh) [inline gui/bricks/MachineInfoBrick.py]: If values_list is a list is has not values() method; if it is a dictionary is should be renamed.,

File: gui/bricks/MotorSpinBoxBrick.py [hunk 5]
```diff
@@ -69,6 +69,7 @@ def __init__(self, *args):
         self.move_step = 1
         self.demand_move = 0
         self.in_expert_mode = None
+        self.pos_history = []
```
Comment (rhfogh) [inline gui/bricks/MotorSpinBoxBrick.py]: Rename to position_history??

File: gui/utils/PropertyBag.py [hunk 6]
```diff
@@ -380,10 +380,12 @@ def __iter__(self):
...
         return len(self.properties)
 
-    def __getitem__(self, propertyKey):
-        item = self.properties.get(propertyKey)
+    def __getitem__(self, property_key):
+        item = self.properties.get(property_key)
+        value = None
         if item is not None:
-            return self.properties[propertyKey].get_value()
+            value = self.properties[property_key].get_value()
```
Comment (rhfogh) [inline gui/utils/PropertyBag.py]: Surely that should be 
'value = item.get_value()'

The following would be shorter (unless pylint protests):

    item = self.properties.get(property_key)
    if item is None:
        return None
    else:
        return item.get_value()

Or even shorter (which I would prefer):

return None if item is None else item.get_value()

**Review by rhfogh — CHANGES_REQUESTED** [review]
Two general comments:
- maybe run black over all the code?
- When adding :param: documentation, as a minimum put in the data type. I would recommend the syntax from the Python 3 typing package. If all you do is repeat the parameter name it is better to have nothing at all

File: gui/bricks/AttenuatorsBrick.py (hunk 2 above)
Comment (IvarsKarpics) [reply gui/bricks/AttenuatorsBrick.py]: I will run autopep and black on all changed files.

File: gui/bricks/EMBL/StillCollectionPreviewBrick.py (hunk 3 above)
Comment (IvarsKarpics) [reply gui/bricks/EMBL/StillCollectionPreviewBrick.py:155]: This is EMBL specific brick and I do not see point to add some checks. Later if this brick becomes a core of mxcube, then one can add a if block.

File: gui/bricks/MachineInfoBrick.py (hunk 4 above)
Comment (IvarsKarpics) [reply gui/bricks/MachineInfoBrick.py]: You are right. I changed it to values_dict

File: gui/bricks/MotorSpinBoxBrick.py (hunk 5 above)
Comment (IvarsKarpics) [reply gui/bricks/MotorSpinBoxBrick.py]: Done

File: gui/utils/PropertyBag.py (hunk 6 above)
Comment (IvarsKarpics) [reply gui/utils/PropertyBag.py]: OK, makes sense

File: gui/bricks/EMBL/StillCollectionPreviewBrick.py (hunk 3 above)
Comment (rhfogh) [reply gui/bricks/EMBL/StillCollectionPreviewBrick.py:155]: Cool.

**Review by rhfogh — APPROVED** [review]
Ready to merge when you say so

**Pull request merged**